docs/concepts-clients.md

```diff
@@ -70,7 +70,7 @@ Messages are received from [brokers](reference-terminology.md#broker) either syn
 
 ### Listener
 
-Client libraries provide listener implementation for consumers. For example, the [Java client](client-libraries-java.md) provides a {@inject: javadoc:MesssageListener:/client/org/apache/pulsar/client/api/MessageListener} interface. In this interface, the `received` method is called whenever a new message is received.
+Client libraries provide listener implementation for consumers. For example, the [Java client](client-libraries-java.md) provides a [MesssageListener](/api/client/org/apache/pulsar/client/api/MessageListener) interface. In this interface, the `received` method is called whenever a new message is received.
```

Comment:
@momo-jun I'm afraid that we have more similar broken links because `@inject: javadoc:` is converted to `/api/client/...` while we removed the files of no version prefixed. Replacing them all by `[...](/api/client/...)` can be a fix.

Reply:
@tisonkun Thanks for this information. //cc @Anonymitaet .

Reply:
Fixed all the broken ones through https://github.com/apache/pulsar-site/pull/547.

Reply:
Thank you!